internal/resourcepaths/siloresourcepaths.go

```diff
@@ -0,0 +1,10 @@
+package resourcepaths
+
+const (
+	// silo container resources paths
+	SiloDeviceResourcePath          string = "Container/Devices/Generic"
+	SiloMappedDirectoryResourcePath string = "Container/MappedDirectories"
+	SiloMappedPipeResourcePath      string = "Container/MappedPipes"
+	SiloMemoryResourcePath          string = "Container/Memory/SizeInMB"
+	SiloRegistryFlushStatePath      string = "Container/RegistryFlushState"
+)
```

Comment:
Any merit in moving the UVM/virtual machine resource paths into this package also?

Reply:
Well the uvm ones aren't used outside of the uvm package, but these are needed to make requests in the tasks. I could see refactoring to remove that last requirement if we want it. What do you think?

Reply:
Just feel like having all of them in one package is easier to reason with

Reply:
I think we should keep it separate for now unless there's a really compelling reason to include them. The concern would be for preventing potential cycles.